identity_base_config/templates/default/id-rails-console.erb

```diff
@@ -40,6 +43,20 @@ if [ $# -ge 1 ]; then
     esac
 fi
 
+while true; do
+    read -r -p "Please provide an explanation for this Rails console: " USE_CASE
+    if [ ! -z "$USE_CASE" ]; then
+        break
+    fi
+done
+
+CURL='curl -X POST --data-urlencode '
+TEXT=":rails:*rails console*:rails: for \`$(logname)\` on \`$(hostname)\`\nuse case: \\\"${USE_CASE}\\\""
+PAYLOAD="'payload={\"channel\": \"<%= @channel %>\", \"username\": \"id-rails-console\", \"text\": \"${TEXT}\", \"icon_emoji\": \":dopetopus:\"}'"
+bash -c "${CURL} ${PAYLOAD} <%= @webhook %>"
```

Comment:
Why do we need to open another bash?

Reply:
This was based off of the additions in `deploy` here: https://github.com/18F/identity-devops/pull/119/files#diff-078f40fa23e0672777adc7c05d4773ddR55-R62 but I can knock it down to just the `curl` command with the `$PAYLOAD`.